src/main/java/module-info.java

```diff
@@ -104,8 +104,6 @@
     // fulltext search
     requires org.apache.lucene.core;
     uses org.apache.lucene.codecs.lucene91.Lucene91Codec;
```

Comment:
```suggestion
    // In case the version is updated, please also adapt SearchFieldConstants#VERSION to the newly used version
    uses org.apache.lucene.codecs.lucene91.Lucene91Codec;
```